nemo/lightning/one_logger_callback.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+OneLogger callback for NeMo training.
+
+This module provides a callback that integrates OneLogger telemetry with NeMo training.
+"""
+from lightning.pytorch import Trainer
+from nv_one_logger.api.config import OneLoggerConfig
+from nv_one_logger.training_telemetry.api.config import TrainingTelemetryConfig
+from nv_one_logger.training_telemetry.api.training_telemetry_provider import TrainingTelemetryProvider
+from nv_one_logger.training_telemetry.integration.pytorch_lightning import TimeEventCallback as OneLoggerPTLCallback
```

Comment:
How much does this callback buy us, it seems to be a very thin wrapper of the training callback, why not use the training callback directly in `OneLoggerNeMoCallback` below, so that we can have `on_app_start()` called in this file?

Reply:
that's great, thanks!